source/content/guides/build-tools/01-introduction.md

```diff
@@ -20,104 +20,77 @@ image: buildToolsGuide-thumb
+- **A Continuous Integration (CI) Service <Popover title="CI Service" content="A Continuous Integration service that provides hosted solutions to run automated tests for a project, allowing them to be run triggered by a change on the Git provider. This is where the source code is turned into production ready code and where automated tests are run." />**
+- **A Pantheon Site**
+  - Create a [free Pantheon account](https://dashboard.pantheon.io/register)
 
-      ```bash
-      terminus auth:login --machine-token=<machine-token>
-      ```
+The supported Git provider and Continuous Integration service combinations are:
+- [<CustomIcon icon="github" /> GitHub](https://github.com) and [<CustomIcon icon="circleci" /> CircleCI](https://circleci.com/)
+- [<CustomIcon icon="gitlab" /> GitLab](https://about.gitlab.com) with [GitLabCI](https://about.gitlab.com/product/continuous-integration/)
+- [<CustomIcon icon="bitbucket" /> BitBucket](https://bitbucket.org/product/) with [BitBucket Pipelines](https://bitbucket.org/product/features/pipelines)
 
-5. Create the `$HOME/.terminus/plugins` directory if it does not already exist:
+Build Tools projects also use [<CustomIcon icon="composer" /> Composer  <Popover title="CI Service" content="Composer is a PHP dependency manager that provides an alternative, more modern way to manage the external code used by a project." />](https://getcomposer.org) for PHP dependency management. If you haven't worked with Composer before, the [Pantheon Composer doc](/composer) is a great place to get started.
 
-      ```bash
-      mkdir -p $HOME/.terminus/plugins
-      ```
+<Enablement title="Automation Training" link="https://pantheon.io/agencies/learn-pantheon?docs">
+Master Composer, automated testing, and other advanced workflow concepts with help from our experts. Pantheon delivers custom workshops to help teams master our platform and improve their [WebOps](https://pantheon.io/webops) workflow.
+</Enablement>
 
-6. Install the [Terminus Composer Plugin](https://github.com/pantheon-systems/terminus-composer-plugin):
+## Build Tools Workflow
+Build Tools projects extend the [Pantheon workflow](https://pantheon.io/docs/pantheon-workflow) by adding an external Git provider and a CI service.
 
-    ```bash
-    composer create-project -n --no-dev -d $HOME/.terminus/plugins pantheon-systems/terminus-composer-plugin:~1
-    ```
+In this workflow, only files unique to the project are tracked as part of external Git repository. The CI service then builds a production artifact, deploys the fully-build site to Pantheon, and runs a suite of automated tests.
 
-7. Install the [Terminus Drupal Console Plugin](https://github.com/pantheon-systems/terminus-drupal-console-plugin):
+A diagram visualizing this workflow with GitHub and CircleCI is below but the concepts are the same when working with other providers.
```

Comment:
@ataylorme would you like is to recreate that image with the tools abstracted?

Reply:
@alexfornuto my plan is to get through rewriting the guide and making it agnostic, then to ask design for help with images